### ISSUE TITLE
Have a connection turn a different colour if there is volume or data transferring currently

### ISSUE THREAD

**Issue by RustoMCSpit** [Houston4444/HoustonPatchbay#15]
Would be useful when you have a lot of stuff and want to quickly find what you're hearing now, especially with a bunch of tabs. At least some sort of visual output is needed, maybe flowing arrows?

**Comment by Houston4444** [comment]
I agree, but it is quite complex to do, and unfortunately, it (probably) needs to create listen port (which we could hide in this app but not in others). In all cases, It needs a lot of work.
